### ISSUE TITLE
VoiceOver unable to read book titles in recently played area.

### ISSUE THREAD

**Issue by rkingett** [rasmuslos/ShelfPlayer#196]
Running the latest version of iOS, VoiceOver, the built in screen reader for Apple devices, doesn't read any of the book titles in the recently played list. VoiceOver only reads the remaining time instead of the book title.

## Steps to reproduce.

1. Enable VoiceOver, then open the app.
2. Flick left or right after touching your finger to the spot where your recently played section is.

[VoiceOver getting started video is here](https://www.youtube.com/watch?v=ROIe49kXOc8)

**Comment by rasmuslos** [comment]
I will look into improving accessibility support in 2.6.0!

**Comment by Neurrone** [comment]
Hey, I've just discovered this app after the upcoming changes to Plex forced me to migrate to Audiobookshelf. I'm also using Voiceover as well and haven't purchased the app yet since I wasn't sure how well it'd work with Voiceover.

Any rough indication of when 2.6 will be ready?

**Comment by rasmuslos** [comment]
VoiceOver improvements haven been released in version 3.0.0

**Comment by math65** [comment]
Thenk you so much !
